modules/xfem/src/dirackernels/XFEMPressure.C

```diff
@@ -8,69 +8,26 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
```

Comment:
Per our discussion, let us follow `Pressure.C` to add `postprocessor` and `factor`.

Reply:
Yes, sir.